### PR TITLE
E-1267 fix: now server switcher not appears on prod

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

File: src/components/screen/Oauth/Oauth.vue [hunk 1]
```diff
@@ -99,6 +102,10 @@ export default {
       return !this.isInited || this.isLoadingContent;
     },
 
+    isServerSwitcherVisible() {
+      return !ENV.VUE_APP_IS_PRODUCTION;
```
Comment (dudiq) [inline src/components/screen/Oauth/Oauth.vue:106]: `VUE_APP_IS_PRODUCTION` is `true` for staging.
solution is not working correctly.

maybe add `VUE_APP_IS_STRICT` flag and set value `true` for prod and staging env files
then `VUE_APP_IS_PRODUCTION` enable only for prod env file.

Comment (ifiri) [reply src/components/screen/Oauth/Oauth.vue:106]: @dudiq is `VUE_APP_IS_PRODUCTION` affects something in `staging`?

Comment (dudiq) [reply src/components/screen/Oauth/Oauth.vue:106]: try to find usage of this const
and yes, for stagin this const is `true`

Comment (ifiri) [reply src/components/screen/Oauth/Oauth.vue:106]: support your idea with `VUE_IS_STRICT`

Comment (ifiri) [reply src/components/screen/Oauth/Oauth.vue:106]: done

**Comment by dudiq** [comment]
@lamartire ready for approve

File: src/components/screen/Oauth/Oauth.vue [hunk 2]
```diff
@@ -7,9 +7,12 @@
     <div v-if="isInited">
       <section class="oauth-section">
         <v-card class="card-content">
+          <logout-button class="oauth-logout" />
           <form-field>
-            <server-switcher @switch="onServerSwitch" />
-            <logout-button class="oauth-logout" />
+            <server-switcher
+              v-if="isServerSwitcherVisible"
```
Comment (dudiq) [inline src/components/screen/Oauth/Oauth.vue:13]: @ifiri move this `v-if="isServerSwitcherVisible"` to parent `<form-field>`.
we don't need soo much empty space
image

it should be like this
image

Comment (ifiri) [reply src/components/screen/Oauth/Oauth.vue:13]: done

**Comment by dudiq** [comment]
@kazurus approved

**Pull request merged**